portal/src/variables.less

```diff
@@ -12,6 +12,8 @@
 @main-font: 'Source Sans Pro', Arial, sans-serif;
 @second-font: 'nunito';
 
+@wide:      ~"only screen and (min-width: 1200px)";
```

Comment:
Kunnen we het niet bij desktop en mobile houden? Of kwam dat echt niet goed uit?

Reply:
dat wilde ik ook wel maar het komt wel echt veel beter uit zo